tsnet/tsnet.go

```diff
@@ -263,7 +275,7 @@ func (s *Server) start() error {
 	s.localAPIListener = lal
 
 	// Override the Tailscale client to use the in-process listener.
```

Comment:
Should this comment be removed since we're no longer overriding global state?

Reply:
There should be a RELNOTE for this as it's a breaking change

Reply:
Done.